hooks/persistence-defectdojo/hook/src/main/java/io/securecodebox/persistence/service/DeduplicationAwaitingService.java

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package io.securecodebox.persistence.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+
+public class DeduplicationAwaitingService {
+    private static final Logger LOG = LoggerFactory.getLogger(DeduplicationAwaitingService.class);
+
+    Integer refetchWaitSeconds;
+
+    public void refetchWait() {
+      try {
+        refetchWaitSeconds = Integer.valueOf(System.getenv("DEFECTDOJO_REFETCH_WAIT_SECONDS"));
```

Comment:
It is good practice in the Java world to extract such a string into a constant (`private static final String`).

I just saw in the code that there are some other `System.getenv()` scattered in the hook's source. That's not good for some reasons:

1. `Syste.getenv()` is a side effect and makes unit testing hard.
2. Such constants and config gathering should be at one place.

I consider to provide a patch to address these two topics.